bindings/cs/Protean-cs/Protean.Net.Tests/TestArray.cs

```diff
@@ -47,6 +47,26 @@ public void TestComparison()
             Assert.AreEqual(1, v3.CompareTo(v2));
         }
 
+        [Test]
+        public void TestHashCode()
+        {
+            TypedArray a1 = new TypedArray(Variant.EnumType.Double, 2);
+            a1[0] = 1.0;
+            a1[1] = 2.0;
+
+            Variant v1 = new Variant(a1);
+
+            TypedArray a2 = new TypedArray(Variant.EnumType.Double, 2);
+            a2[0] = 1.0;
+            a2[1] = 2.0;
+
+            Variant v2 = new Variant(a2);
+
+            Assert.AreEqual(v1, v2);
+            Assert.AreNotSame(v1, v2);
+            Assert.AreEqual(v1.GetHashCode(), v2.GetHashCode());
```

Comment:
These are fine, but I wonder for these tests if there's any reason not to hard-code the expected hash value too. I don't think that it should ever change, or that there's any reason why it should be non-deterministic.

Reply:
Not sure what we would gain from that other than the ability to test that some hashes don't collide.